lib/parser/init-transform.js

```diff
@@ -0,0 +1,18 @@
+const fs = require('fs');
+const path = require('path');
+const transform = require('./transformations').transform;
+
+module.exports = function initTransform(opts) {
+	global.options = opts;
+	const templateFP = path.join(__dirname, 'utils', 'template.js');
+	const template = fs.readFileSync(templateFP, 'utf8');
```

Comment:
It looks like you never remove the tmp file. You might want to take a look at https://www.npmjs.com/package/tmp to automate this.